supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesPipelineInfo.scala

```diff
@@ -5,12 +5,12 @@ import wdl4s.parser.MemoryUnit
 
 import scala.collection.JavaConverters._
 
-sealed trait JesRuntimeInfo {
+sealed trait JesPipelineInfo {
```

Comment:
Google refers to the pipeline as "create time" and the run itself as "runtime", and what this is is the resources for the pipeline, which was confusing.